### PR TITLE
changed up api for news

### PR DESCRIPTION
**Pull request by napa-chris**
Here are some proposed changes for the api news sdk

### CODE REVIEW

**Comment by napa-chris** [comment]
I think I can probably start splitting those extra classes out, at the bottom. Or renaming them some of them so anyone using it will know that's its still apart of the AlchemyAPI just by name. Entity, for example, is pretty generic.

**Comment by charliieuy** [comment]
First pull request approval!

File: napa_app/napa_app/sdk/AlchemyAPI_NewsParams.cs [hunk 1]
```diff
@@ -7,131 +7,89 @@ namespace AlchemyAPI
...
-        private const string TAXONOMY_LABEL = "&q.enriched.url.enrichedTitle.taxonomy.taxonomy_.label=";
-        private const string CONCEPT_TEXT = "&q.enriched.url.concepts.concept.text=";
-        private const string KEYWORD_TEXT = "&q.enriched.url.enrichedTitle.keywords.keyword.text=";
-        private const string ENTITY_TEXT = "&q.enriched.url.enrichedTitle.entities.entity.text=";
-        private const string ENTITY_TYPE = "&q.enriched.url.enrichedTitle.entities.entity.type=";
-        private const string RELATION = "&q.enriched.url.enrichedTitle.relations.relation=";
-        private const string SENTIMENT = "&q.enriched.url.enrichedTitle.docSentiment=";
-        private const string SENTIMENT_POSITIVE = "type=positive,";
-        private const string SENTIMENT_NEGATIVE = "type=negative,";
+        private readonly Dictionary<string, string> _commands;
```
Comment (charliieuy) [inline napa_app/napa_app/sdk/AlchemyAPI_NewsParams.cs:21]: This is a far better way of adding more data points..

File: napa_app/napa_app/sdk/AlchemyAPI_NewsParams.cs [hunk 2]
```diff
@@ -7,131 +7,89 @@ namespace AlchemyAPI
...
+    public EntityType Type { get; set; }
+    public string Text { get; set; }
+    public Entity(EntityType type, string text)
+    {
+        Type = type;
+        Text = text;
+    }
+}
+
+public enum SentimentType
```
Comment (charliieuy) [inline napa_app/napa_app/sdk/AlchemyAPI_NewsParams.cs:85]: Since this is just a true or false, would we need an enum to represent that?

Comment (napa-chris) [inline napa_app/napa_app/sdk/AlchemyAPI_NewsParams.cs:85]: Which one did you mean? Sentiment type? If so I think it's actually Negative, Neutral, Positive.
If you meant EntityType, I think those vary quite a bit.

**Pull request merged**